src/main/resources/standardTestCases/tmp_dir/tmp_dir.wdl

```diff
@@ -0,0 +1,12 @@
+workflow tmp_dir {
+    call mkTmpDir
+}
+
+task mkTmpDir {
+    command {
+        echo "tmp_dir test wdl" > tmp
```

Comment:
Do we also want to test the temp dir itself?

`echo "tmp_dir test wdl 2" > $TMPDIR/tmp`